python/test/utils/spark_test_case.py

```diff
@@ -11,16 +13,13 @@ class SparkTestCase(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls) -> None:
-        cls.library_location = f"mosaic/lib/mosaic-{version('databricks-mosaic')}-jar-with-dependencies.jar"
+        cls.library_location = f"{mosaic.__path__[0]}/lib/mosaic-{version('databricks-mosaic')}-jar-with-dependencies.jar"
```

Comment:
@sllynn this changes depending on how the wheel is installed? Or we need to try catch here?

Reply:
No, this is just for local / gh tests.